citi-onboarding/mural-valores
# Feat/app base

## What I did

- Create project base

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout feat/app-base`
   1. `git pull`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000/<route>

1. Check if the application runs
